comp/core/log/logger.go

```diff
@@ -139,6 +157,12 @@ func (*logger) Criticalf(format string, params ...interface{}) error {
 	return log.CriticalStackDepth(2, fmt.Sprintf(format, params...))
 }
 
+// GetLogLevel implements Component#GetLogLevel.
+func (*logger) GetLogLevel() (Level, error) {
```

Comment:
Shall we test `GetLogLevel()` for each level, the risk is low, but testing that is cheap also.

Not blocker, but I think it's good to have higher coverage.